.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,19 @@
+name: Trigger DynamicPipelineMakerLambda
+
+on:
+  pull_request:
+    types: [opened, reopened, closed]
+
+jobs:
+  dynamic-pipeline-maker-lambda:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger DynamicPipelineMakerLambda
+        id: invoke
+        uses: gagoar/invoke-aws-lambda@master
+        with:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          REGION: eu-west-1
+          FunctionName: ${{ secrets.AWS_CICD_FUNCTION_NAME }}
+          Payload: ${{ toJSON(github) }}
```

Comment:
Only question: Is this a Github actions function?

Reply:
Yes